felix/calc/l3_route_resolver.go

```diff
@@ -561,7 +574,7 @@ func (c *L3RouteResolver) onNodeUpdate(nodeName string, newNodeInfo *l3rrNodeInf
 	if newNodeInfo != nil {
 		c.nodeNameToNodeInfo[nodeName] = *newNodeInfo
 		for _, a := range newNodeInfo.AddressesAsCIDRs() {
-			c.trie.AddHost(a, nodeName)
+			c.trie.AddHost(a, nodeName, newNodeInfo)
```

Comment:
Sorry if this is obvious, but why the switch to using nodeinfo here instead of just node name?

Reply:
The goal of this patch is to add some information to the message in Felix, so using nodeInfo allows to pass in data like: `ASNumber`, `WireguardPublicKey`,  `Labels`.